CitiesHarmony.IPatch/CMPatch.cs

```diff
@@ -0,0 +1,80 @@
+﻿using Mono.Cecil;
+using Mono.Cecil.Cil;
+using Patch.API;
+using System;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+
+namespace CitiesHarmony.API {
+    public class CMPatch : IPatch {
+        public int PatchOrderAsc { get; } = 1000;
```

Comment:
very high patch order number means that this patch will be applied after all others.